networking/ingress-controller-dnsmgt.adoc

```diff
@@ -0,0 +1,40 @@
+:_content-type: ASSEMBLY
+[id="ingress-controller-dnsmgt"]
+= Configuring an Ingress Controller for manual DNS Management
+include::_attributes/common-attributes.adoc[]
+:context: ingress-controller-dnsmgt
+
+toc::[]
+
+As a cluster administrator, when you create an Ingress Controller, the Operator manages the DNS records automatically. This has some limitations when the required DNS zone is different from the cluster DNS zone or when the DNS zone is hosted outside the cloud provider.
+
+As a cluster administrator, you can configure an Ingress Controller to stop automatic DNS management and start manual DNS management. Set `dnsManagementPolicy` to specify when it should be automatically or manually managed.
+
+== `Managed` DNS management policy
+The `Managed` DNS management policy for Ingress Controllers ensures that the lifecycle of the wildcard DNS record on the cloud provider is automatically managed by the Operator.
+
+During transitioning an Ingress Controller from `Managed` to `Unmanaged` DNS management policy, the Operator does not clean up the previous wildcard DNS record provisioned on the cloud.
+During transitioning an Ingress Controller from `Unmanaged` to `Managed` DNS management policy, the Operator attempts to create the DNS record on the cloud provider if it does not exist or updates the DNS record if it already exists.
+
+== `Unmanaged` DNS management policy
+The `Unmanaged` DNS management policy for Ingress Controllers ensures that the lifecycle of the wildcard DNS record on the cloud provider is not automatically managed, instead it becomes the responsibility of the cluster administrator.
+
+[NOTE]
+====
+On the AWS cloud platform, if the domain on the Ingress Controller does not match with `dnsConfig.Spec.BaseDomain` then the DNS management policy is automatically set to `Unmanaged`.
+====
+
+[IMPORTANT]
+====
+When you set `dnsManagementPolicy` to `unmanaged`, you have to manually manage the lifecycle of the wildcard DNS record on the cloud provider. 
+====
```

Comment:
Can we put this `IMPORTANT` in a very noticeable place, at the top before they read (or skip) any other paragraphs?

Reply:
I have placed it in the overview